lib/ReactViews/Map/Panels/SharePanel/ShareUrl/ShareUrlBookmark.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import { useTranslation } from "react-i18next";
+
+import ViewState from "../../../../../ReactViewModels/ViewState";
+
+import Text from "../../../../../Styled/Text";
+
+interface IShareUrlBookmarkProps {
+  viewState: ViewState;
+}
+
+const bookMarkHelpItemName = "bookmarkHelp";
+
+export const ShareUrlBookmark: FC<IShareUrlBookmarkProps> = ({ viewState }) => {
```

Comment:
Would we ever use this component outside of `SharePanel` (or `SharePanelContent` as it is currently)? I initially has this as a separate component too but may unnecessarily complicate the structure.

Reply:
I would leave it like this to separate logic that gets the bookmark help button from help items from the share panel, someone who forks the repo then can just remove the usage of bookmark but functionality is there or decide to add something else (fewer conflicts on future merge his change don't depend on something else i.e. we changed how help items work and it is no more item name but itemId)